testdata/sqllogictests/functions/csv_scan.slt

```diff
@@ -33,7 +33,7 @@ select count(*) from csv_scan(
 
 # Relative path
 query I
-select count(*) from csv_scan('../../testdata/sqllogictests_datasources_common/data/bikeshare_stations.csv')
+select count(*) from csv_scan('${PWD}/testdata/sqllogictests_datasources_common/data/bikeshare_stations.csv')
```

Comment:
Some of these tests purposely use relative path segments to ensure we work with both relative and absolute paths.

Reply:
hmm... I'll have to think on that one a bit. Making it into a binary forced it to not use a fixed path, but instead use one passed in from the command line.

Reply:
@scsmithr This should be resolved now. It uses the repo root instead of the path from `crates/testing/`, which is arguably better anyways.